c7n/resources/dms.py

```diff
@@ -18,8 +18,14 @@
 from c7n.actions import BaseAction
 from c7n.manager import resources
 from c7n.query import QueryResourceManager, DescribeSource
-from c7n.utils import local_session, chunks, type_schema
+from c7n.utils import local_session, chunks, type_schema, get_retry
 from c7n.filters.vpc import SecurityGroupFilter, SubnetFilter
+from c7n.filters import FilterRegistry
+from c7n.tags import RemoveTag, Tag, TagActionFilter, TagDelayedAction
+
+
+filters = FilterRegistry('dms.filters')
```

Comment:
we should really just leave this as the class variable, there will be other resources here when dms resources get flushed out. you can still do tag action registration after the class is defined using the default filter registry on the class, but then the usage is qualified and explicit instead of a module level 'filters' which is only associated implicitly which is confusing when there are multiple resources in a module.

Reply:
and that makes a lot of sense now that you mention it... ill go back & make similar changes to the sagemaker module